app/views/qsets/show_questions.html.erb

```diff
@@ -6,8 +6,11 @@
     <%= render "question_page_header_small_screen" %>
   </div>
   <div class="row">
-    <div class="col-sm-12">
-      <%= render "question_list" %>
+    <div class="col-sm-12 show-large-question">
+      <%= render "question_list_large_screen" %>
+    </div>
+    <div class="col-sm-12 show-small-question">
+      <%= render "question_list_small_screen" %>
```

Comment:
Is there a way to still do this without repeating the div's in question_list_large_screen and question_list_small_screen?